spec/controllers/v2/security_guides_controller_spec.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
From where this name is coming from? Shouldn't this be `spec_helper`? Would it work without it?